config/environments/production.rb

```diff
@@ -80,14 +80,33 @@
   # require 'syslog/logger'
   # config.logger = ActiveSupport::TaggedLogging.new(Syslog::Logger.new 'app-name')
 
-  if true # ENV["RAILS_LOG_TO_STDOUT"].present?
+  if ENV["RAILS_LOG_TO_STDOUT"].present?
```

Comment:
`RAILS_LOG_TO_STDOUT`should be defined in the Docker file to avoid change the behavior on integration tests and test clusters

`ENV RAILS_LOG_TO_STDOUT=true`